app/assets/stylesheets/user_i_ds.css

```diff
@@ -0,0 +1,4 @@
+/*
```

Comment:
It is also worth noting that class names should be underscored rather than camel case when generating.